client/containers/DashboardOverview/overviewRows/PubOverviewRow.tsx

```diff
@@ -19,7 +19,7 @@ type Props = {
 	labels?: IconLabelPair[];
 	className?: string;
 	pub: Pub;
-	inCollection?: boolean;
+	inCollection?: Collection;
```

Comment:
The changes in this file fix a small(?) bug where Pub links from Collection overviews don't have a `collectionSlug`, and we lose the Collection information from the Pub dashboard breadcrumbs.